### PR TITLE
Registartion works now

### PR DESCRIPTION
**Pull request by DanNelson99**
Fixed up registration to work with formik

### CODE REVIEW

File: src/registration/Registration.jsx [hunk 1]
```diff
@@ -27,78 +33,159 @@ class Registration extends React.Component {
...
             <div className={classes.root}>
-            <Form onSubmit={register} form={FORM_NAME} submitLabel={t('register:register')}>
-                <Grid 
-                     container
-                     direction="column"
-                     justify="center"
-                     alignItems="flex-end"
+                {/* <Form onSubmit={register} form={FORM_NAME} submitLabel={t('register:register')}> */}
+                <Grid container direction='column' justify='flex-start' alignItems='center'>
+                    <h1>Registration</h1>
```
Comment (hpowell20) [inline src/registration/Registration.jsx]: Can the labels be added to resource files?

File: src/registration/Registration.jsx [hunk 2]
```diff
@@ -27,78 +33,159 @@ class Registration extends React.Component {
...
+                                            placeholder={t('register:orgName')}
+                                        />
+                                    </Grid>
+                                    <Grid className={classes.field} item>
+                                        <Field
+                                            required={true} component={TextField} type="text"
+                                            name="email" label={t('register:email')}
+                                            margin="normal"
+                                            variant="outlined"
+                                            placeholder={t('register:orgName')}
```
Comment (hpowell20) [inline src/registration/Registration.jsx]: Should this placeholder name be email as opposed to orgName?

File: src/registration/Registration.jsx [hunk 3]
```diff
@@ -27,78 +33,159 @@ class Registration extends React.Component {
...
+                                            required={true} component={TextField} type="text"
+                                            name="adminEmail" label={t('register:email')}
+                                            margin="normal"
+                                            variant="outlined"
+                                            placeholder={t('register:email')}
+                                            className={classes.textField}
+                                        />
+                                    </Grid>
+                                    <Grid className={classes.field} item>
+                                        <Field
```
Comment (hpowell20) [inline src/registration/Registration.jsx]: The password is being automatically generated once the organization has been verified.  The only information we are asking for for an admin user is first name, last name, and email

**Comment by DanNelson99** [comment]
I got the form validation working. I can't seem to get i18n using the resources files. So I set the default value for now. I will have to go back and fix this. I Also need the final list of fields @hpowell20. did we agree on city and province  with address containing street.

**Comment by DanNelson99** [comment]
screen shot 2019-01-04 at 5 37 01 pm
screen shot 2019-01-04 at 5 37 11 pm

**Pull request merged**